crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -0,0 +1,381 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use crate::{cli, cli::traits::Cli as _, cli::Cli, style::style};
+use clap::{Args, ValueEnum};
+use cliclack::{confirm, input};
+use pop_common::Profile;
+use pop_parachains::{
+	binary_path, build_parachain, export_wasm_file, generate_genesis_state_file,
+	generate_plain_chain_spec, generate_raw_chain_spec, is_supported, replace_chain_type,
+	replace_protocol_id, replace_relay_spec,
+};
+use std::{env::current_dir, fs::create_dir_all, path::PathBuf};
+#[cfg(not(test))]
+use std::{thread::sleep, time::Duration};
+use strum::{EnumMessage, VariantArray};
+use strum_macros::{AsRefStr, Display, EnumString};
+
+const DEFAULT_SPEC_NAME: &str = "template-chain-spec.json";
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Default,
+	Debug,
+	Display,
+	EnumString,
+	EnumMessage,
+	ValueEnum,
+	VariantArray,
+	Eq,
+	PartialEq,
+)]
+/// Supported chain types for the resulting chain spec.
+pub(crate) enum ChainType {
+	// A development chain that runs mainly on one node.
+	#[default]
+	#[strum(
+		serialize = "Development",
+		message = "Development",
+		detailed_message = "Meant for a development chain that runs mainly on one node."
+	)]
+	Development,
+	// A local chain that runs locally on multiple nodes for testing purposes.
+	#[strum(
+		serialize = "Local",
+		message = "Local",
+		detailed_message = "Meant for a local chain that runs locally on multiple nodes for testing purposes."
+	)]
+	Local,
+	// A live chain.
+	#[strum(serialize = "Live", message = "Live", detailed_message = "Meant for a live chain.")]
+	Live,
+}
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Default,
+	Debug,
+	Display,
+	EnumString,
+	EnumMessage,
+	ValueEnum,
+	VariantArray,
+	Eq,
+	PartialEq,
+)]
+/// Supported relay chains that can be included in the resulting chain spec.
+pub(crate) enum RelayChain {
```

Comment:
Can we reconsider the order of the variants please? A suggestion is ordering by likelihood of selection, or the natural progression - i.e. Paseo/Westend to Kusama to Polkadot.